.github/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+name-template: "$RESOLVED_VERSION"
```

Comment:
We would rather prefer to use this action [here](https://github.com/EddieHubCommunity/LinkFree/blob/main/.github/workflows/release.yml)